Packs/QuestKace/Integrations/QuestKace/QuestKace.yml

```diff
@@ -0,0 +1,484 @@
+      description: Video controlleres of the machine.
+      type: String
+    - contextPath: QuestKace.Machines.Monitor
+      description: Monitor of the machine.
+      type: String
+    - contextPath: QuestKace.Machines.RegistrySize
+      description: Size of Registory of the machine.
+      type: String
+    - contextPath: QuestKace.Machines.RegistryMaxSize
+      description: Max Size of registry of the machine.
+      type: String
+    - contextPath: QuestKace.Machines.PagefileSize
+      description: Size of pagefile of the machine.
+      type: String
+    - contextPath: QuestKace.Machines.PagefileMaxSize
+      description: Max size of pagefile of the machine.
+      type: String
+    - contextPath: QuestKace.Machines.ManualEntry
+      description: Number of Manual entry to the machine.
+      type: String
+  - deprecated: false
+    description: All assets of QuestKace system.
+    execution: false
+    name: kace-assets-list
+    outputs:
+    - contextPath: QuestKace.Assets.ID
+      description: ID of the asset.
+      type: Number
+    - contextPath: QuestKace.Assets.AssetTypeId
+      description: Type ID of the asset.
+      type: Number
+    - contextPath: QuestKace.Assets.Name
+      description: Name of the asset.
+      type: String
+    - contextPath: QuestKace.Assets.AssetDataId
+      description: Data ID of the asset.
+      type: Number
+    - contextPath: QuestKace.Assets.OwnerId
+      description: Owner ID of the asset.
+      type: Number
+    - contextPath: QuestKace.Assets.Modified
+      description: Modified date of the asset.
+      type: String
+    - contextPath: QuestKace.Assets.Created
+      description: Created date of the asset.
+      type: String
+    - contextPath: QuestKace.Assets.MappedId
+      description: Mapped ID of the asset.
+      type: Number
+    - contextPath: QuestKace.Assets.AssetClassId
+      description: Class ID of the asset.
+      type: Number
+    - contextPath: QuestKace.Assets.Archieve
+      description: Archive of the asset.
+      type: String
+    - contextPath: QuestKace.Assets.AssetStatusId
+      description: Status ID of the asset.
+      type: Number
+    - contextPath: QuestKace.Assets.AssetTypeName
+      description: Type name of the asset.
+      type: String
+  - deprecated: false
+    description: List of all queues that are in QuestKace system.
+    execution: false
+    name: kace-queues-list
+    outputs:
+    - contextPath: QuestKace.Queues.ID
+      description: ID of the queue
+      type: Number
+    - contextPath: QuestKace.Queues.Name
+      description: Name of the queue
+      type: String
+  - arguments:
+    - default: false
+      defaultValue: '"hd_ticket all,submitter limited,owner limited, asset limited,machine
+        limited,priority limited,category limited, impact limited,status limited,
+        related_tickets limited"'
+      description: Custom fields for shaping tickets list
+      isArray: false
+      name: custom_fields
```

Comment:
is it a `custom_query` or `custom_fields` that you add to the query? I'm a bit puzzled.

Reply:
Changed it to custom_shaping in kace-tickets-list
In create and update the it is custom_fields.